server/offenceCodeDecisions/DecisionTree.ts

```diff
@@ -1,171 +1,123 @@
-        [Role.ATTEMPTED, `Who did ${Text.OFFENDER_FULL_NAME} attempt to detain?`],
-        [Role.INCITED, `Who did ${Text.OFFENDER_FULL_NAME} incite another prisoner to detain?`],
-        [Role.ASSISTED, `Who did ${Text.OFFENDER_FULL_NAME} assist ${Text.ASSISTED_FULL_NAME} to detain?`],
-      ])
-      .child(decision('Another prisoner').type(Type.PRISONER).code(2001))
-      .child(decision('A prison officer').type(Type.OFFICER).code(2002))
-      .child(decision('A member of staff who is not a prison officer').type(Type.STAFF).code(2003))
-      .child(decision('Another person not listed above').type(Type.OTHER_PERSON).code(2004))
-  )
-  .child(
-    decision('Stopping someone who is not a prisoner from doing their job')
-      .title('What did the incident involve?')
-      .child(decision('Denying someone access to any part of the prison').code(3001))
-      .child(decision('Obstructing a member of staff from doing their job').code(6001))
-      .child(
-        decision('Stopping someone carrying out a drug test')
-          .title('What happened?')
-          .child(decision('Tampering with or falsifying a drug testing sample').code(23201))
-          .child(decision('Refuses to provide a sample for drug testing').code(23202))
-      )
-  )
-  .child(
-    decision('Being absent without authorisation, being in an unauthorised place, or failing to work correctly')
-      .title('What did the incident involve?')
-      .child(decision('Being absent without authorisation').code(18001))
-      .child(decision('Being in an unauthorised place').code(18002))
-      .child(decision('Failing to work correctly').code(21001))
-  )
+export default decision([
+  [Role.COMMITTED, `What type of offence did ${Text.PRISONER_FULL_NAME} commit?`],
+  [Role.ATTEMPTED, `What type of offence did ${Text.PRISONER_FULL_NAME} attempt to commit?`],
+  [Role.ASSISTED, `What type of offence did ${Text.PRISONER_FULL_NAME} assist another prisoner to commit?`],
+  [Role.INCITED, `What type of offence did ${Text.PRISONER_FULL_NAME} incite another prisoner to commit?`],
+]).child(
+  answer('Assault, fighting, or endangering the health or personal safety of others').child(
+    decision('What did the incident involve?').child(
+      answer('Assaulting someone').child(
+        decision([
+          [Role.COMMITTED, 'Who was assaulted?'],
+          [Role.ATTEMPTED, `Who did Who did ${Text.PRISONER_FULL_NAME} attempt to assault?`],
+          [Role.ASSISTED, `Who did Who did ${Text.PRISONER_FULL_NAME} assist ${Text.ASSOCIATED_PRISONER_FULL_NAME} to assault?`,],
+          [Role.INCITED, `Who did ${Text.PRISONER_FULL_NAME} incite another prisoner to assault?`],
+        ]).child(
+          answer(['Another prisoner', `Another prisoner - ${Text.VICTIM_PRISONER_FULL_NAME}`]).type(Type.PRISONER).child(
+            decision('Was the incident a racially aggravated assault?').child(
+              answer('Yes').offenceCode(1001)).child(
```

Comment:
Nots rue if this is the source of your linting issues, but this format doesn't seem to make sense (unless I am missing something)..

each option is a child of another? The previous version was better format-wise - I guess there is a technical reason for doing it this way?